asa_parser/src/asa_parse.py

```diff
@@ -205,40 +205,40 @@ def show_cpu_detailed(self):
         return json.dumps({'text': self.get_show_section('cpu detailed')})
 
     def ipsec_stats(self):
-        """Parser for show ipsec stats"""
```

Comment:
It looks like you are changing the indentation on ipsec_stats() here - why is that?

Reply:
Oh sorry. I had a section commented out because it was giving errors.